### PR TITLE
AT_14.03.02 | <MCPr> Verify possibility to edit description

### PR DESCRIPTION
**Pull request by JuliaP2023**
https://trello.com/c/mj6PQXh2/2064-at140302-multiconfiguration-project-verify-possibility-to-edit-description

### CODE REVIEW

File: cypress/pageObjects/MultiConfigurationProjectPage.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ class MultiConfigurationProjectPage {
     getDescriptionInputField = () => cy.get('#description textarea.jenkins-input');
     getSaveDescriptionBtn = () => cy.get('button.jenkins-button.jenkins-button--primary ');
     getRenameBtnMultiConfPrj = () => cy.get('a[href*="rename"]')
+    getEditDescriptionBnt = () => cy.get('a#description-link.jenkins-button--tertiary');
```
Comment (Rom-T) [inline cypress/pageObjects/MultiConfigurationProjectPage.js:21]: I would change the name of the element(getAddDescriptionBtn) and method(clickAddDescriptionBtn) to getAddEditDescriptionBtn/clickAddEditDescriptionBtn so as not to duplicate the locators. And of course I corrected all the tests where they are used

**Comment by S-Lanchik** [comment]
Please add PR link to the trello card

**Comment by JuliaP2023** [comment]
[AT_14.03.02 | <Multiconfiguration project> Verify possibility to edit description](https://trello.com/c/mj6PQXh2/2064-at140302-multiconfiguration-project-verify-possibility-to-edit-description)

**Comment by JuliaP2023** [comment]
[AT_14.03.02 | <Multiconfiguration project> Verify possibility to edit description](https://trello.com/c/mj6PQXh2/2064-at140302-multiconfiguration-project-verify-possibility-to-edit-description)

**Comment by JuliaP2023** [comment]
[AT_14.03.02 | <Multiconfiguration project> Verify possibility to edit description](https://trello.com/c/mj6PQXh2/2064-at140302-multiconfiguration-project-verify-possibility-to-edit-description)

File: cypress/pageObjects/MultiConfigurationProjectPage.js (hunk 1 above)
Comment (JuliaP2023) [reply cypress/pageObjects/MultiConfigurationProjectPage.js]: I have changed as recommended, the AT_14.03.01 I will change through RF card in Trello, as the test has been already merged

File: cypress/e2e/tests/multiConfigurationProject.cy.js [hunk 2]
```diff
@@ -130,4 +130,20 @@ describe("multiConfigurationProject", () => {
     })
  }) 
 
+
+    it('AT_14.03.02 | <Multi-configuration project> Verify possibility to edit description by clicking Edit Description', () => {
+        cy.createMultiConfigurationProject(newItemPage.multiConfigurationProjectName); 
+                
+        homePage
+            .clickMultiConfigProjectNameLink(newItemPageData.multiConfigurationProjectName)
+            .clickAddDescriptionBtn()
```
Comment (Rom-T) [inline cypress/e2e/tests/multiConfigurationProject.cy.js:139]: Change please to .clickAddEditDescriptionBtn()

Comment (JuliaP2023) [reply cypress/e2e/tests/multiConfigurationProject.cy.js]: done

**Review by SiarheiMSV — APPROVED** [review]

File: cypress/e2e/tests/multiConfigurationProject.cy.js [hunk 3]
```diff
@@ -115,4 +115,23 @@ describe("multiConfigurationProject", () => {
...
+            .typeDescriptionInputField(multiConfigurationProjectConfigurePage.descriptionText)
+            .clickSaveDescriptionBtn()
+            .clickEditDescriptionBtn()
+            .typeDescriptionInputField(multiConfigurationProjectConfigurePage.newDescriptionText)
+            .clickSaveDescriptionBtn()
+            .getDescriptionField().should('have.text', multiConfigurationProjectConfigurePage.newDescriptionText)
```
[the remark this answers is not in the document]
Comment (JuliaP2023) [reply cypress/e2e/tests/multiConfigurationProject.cy.js]: Done

File: cypress/pageObjects/MultiConfigurationProjectPage.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ class MultiConfigurationProjectPage {
     getAddDescriptionBtn = () => cy.get('a#description-link.jenkins-button.jenkins-button--tertiary');
     getDescriptionInputField = () => cy.get('#description textarea.jenkins-input');
     getSaveDescriptionBtn = () => cy.get('button.jenkins-button.jenkins-button--primary ');
+    getEditDescriptionBnt = () => cy.get('a#description-link.jenkins-button--tertiary');
```
[the remark this answers is not in the document]
Comment (JuliaP2023) [reply cypress/pageObjects/MultiConfigurationProjectPage.js]: Уже переименовала для этого теста строка 21,  у Романа было это замечание, дубликат со строки 17 уберу когда буду рефакторить тест AT_14.03.01, потому что он там используется сейчас

**Pull request merged**